src/main.tex

```diff
@@ -11,6 +11,9 @@
 \usepackage{subfigure}
 \usepackage{makeidx}
 \usepackage{multicol}
+\usepackage{standalone}
```

Comment:
Let's include a blank line and/or a comment here to keep track of packages we're adding ourselves (as opposed to the ones the publisher needs).

Reply:
(Doing my own diagram now and noticed this.)

Reply:
I've gone with:

```

\renewcommand{\sfdefault}{txss}

\usepackage{changepage}% for text width to include margin

\usepackage{customenvironments}

% Adding our own packages
\usepackage{standalone}
\usepackage{tikz}
\usetikzlibrary{arrows}
%---------------------

```